src/install_update.ts

```diff
@@ -66,15 +66,22 @@ export async function createUpdateResp(
 
   try {
     const cwd = Deno.cwd();
-    const dependencyFiles = await gatherDependencyFiles(cwd);
-    let updateResponses: Update[] = [];
+    const { dependencyFiles } = await gatherDependencyFiles(cwd);
 
     for (const [file, _] of dependencyFiles) {
       // Update dependency file with latest dependency versions
-      const updateResp = await updateDependencyFile(`${cwd}/${file}`, releases);
-      updateResponses = [...updateResponses, ...updateResp];
+      try {
+        const fileUpdateResp = await updateDependencyFile(
+          `${cwd}/${file}`,
+          releases,
+        );
+        updateResp.updates = [...updateResp.updates, ...fileUpdateResp];
+      } catch (err) {
+        updateResp.error = updateResp.error
+          ? { message: updateResp.error.message += `\n   ${err.message}` }
```

Comment:
nit: What do we get from the `+=` here rather than just a `+`? I'm not sure there's a functional difference in this case.

Reply:
Oh, probably the same. I think I was just in the mode of `+=` because of the other concat'ing that was being done.